dataland-frontend/index.html

```diff
@@ -4,7 +4,6 @@
         <script id="Cookiebot"
                 src="https://consent.cookiebot.com/uc.js"
                 data-cbid="cba5002e-6f0e-4848-aadc-ccc8d5c96c86"
-                data-blockingmode="auto"
                 type="text/javascript"></script>
```

Comment:
Based on documentation my understanding is that we need a `async` statement here.

https://support.cookiebot.com/hc/en-us/articles/360009074960-Automatic-cookie-blocking